CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## 1.4.0
+
+* [Change crate layout to current practice](https://github.com/frewsxcv/rust-threadpool/pull/64)
+
```

Comment:
The users of this library don't need to know about this change since it doesn't affect them, so we don't need a changelog entry